docs/sources/flow/install/kubernetes.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Kubernetes
+weight: 100
+---
+
+## Install Grafana Agent Flow on Kubernetes
+
+Grafana Agent Flow can be installed on Kubernetes by using the Helm chart for
+Grafana Agent Flow.
+
+## Before you begin
+
+* Ensure that you have [Helm][] installed on your system.
+* Ensure that you have a Kubernetes cluster to deploy Grafana Agent Flow to.
+* Ensure that your local Kubernetes context is configured to point to the
+  correct cluster.
+
+[Helm]: https://helm.sh
+
+## Steps
+
+> **NOTE**: These instructions install the generic Helm chart for Grafana
+> Agent, which installs Grafana Agent Flow by default.
```

Comment:
I'm not sure what 'generic' adds here. Also, could we have a link to where the chart is on our repo so people can poke in and maybe contribute?